bentoml/_internal/server/__init__.py

```diff
@@ -71,20 +74,13 @@ def _start_ngrok_server() -> None:
 
 def serve_production(
     bento_path_or_tag: str,
-    working_dir: Optional[str] = None,
-    port: Optional[int] = None,
+    working_dir: t.Optional[str] = None,
+    app_workers: t.Optional[int] = None,
```

Comment:
not related to this PR, I was thinking about how should we expose the `workers` option to end-user and thought it  would be helpful to name them a bit more specific, `api_worders` and `runner_workers`

Reply:
on a related note, `workers` should be limited to 1 for development mode

Reply:
IMO we should only expose this in the configuration file, allows users to config workers(or quota) for api server and each runner.